### DISCUSSION TITLE
Level condition of where function

### DISCUSSION THREAD

**Discussion by alexis-bvq** [atoti/atoti#645] [General]
## Description
When we type a condition on a level such as `lvl["l1"] != None"` in order to not have a total value, we can not write `lvl["l1"] is not None`. This leads to pylint error/warning C0121.  Same warning appears when you try a string or boolean condition on a level.

**Comment by tibdex** [comment]
[Pandas does something similar](https://stackoverflow.com/questions/62650695/how-can-i-make-filtering-a-pandas-dataframe-with-a-condition-conform-to-pep8). I would recommend disabling this Pylint rule as [we do in the project template](https://stackoverflow.com/questions/62650695/how-can-i-make-filtering-a-pandas-dataframe-with-a-condition-conform-to-pep8).
